### PR TITLE
audio: add targetBitrate stats

### PR DESCRIPTION
**Pull request by fippo**
https://bugs.chromium.org/p/webrtc/issues/detail?id=13377

### CODE REVIEW

**Review by alvestrand — APPROVED** [review]
Nice to change the internal delta from msec to sec :-)

**Comment by fippo** [comment]
I ended up being very confused when I only saw a blue line :-)

File: src/content/peerconnection/audio/js/main.js [hunk 1]
```diff
@@ -356,11 +360,12 @@ window.setInterval(() => {
           // append to chart
           bitrateSeries.addPoint(now, bitrate);
           headerrateSeries.addPoint(now, headerrate);
-          bitrateGraph.setDataSeries([bitrateSeries, headerrateSeries]);
+          targetBitrateSeries.addPoint(now, report.targetBitrate);
+          bitrateGraph.setDataSeries([bitrateSeries, headerrateSeries, targetBitrateSeries]);
```
Comment (alvestrand) [inline src/content/peerconnection/audio/js/main.js:364]: when running on pre-bitrate-stat browsers: will addPoint(now, null) work fine and do nothing?

Comment (fippo) [reply src/content/peerconnection/audio/js/main.js:364]: Yes. The graph library interprets "undefined" as "no value" and does not draw it.

**Review by KaptenJansson — APPROVED** [review]
LGTM

**Pull request merged**